docs/example-scenario/data/data-warehouse.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Building an enterprise data warehouse from multiple data sources
```

Comment:
Given the feedback during our meeting, how about: "Data warehousing and analytics for sales and marketing" ?

Reply:
Agree